### ISSUE TITLE
SetInt & SetGlobalInt

### ISSUE THREAD

**Issue by MamaCatDev** [Unity-Technologies/PostProcessing#689]
##### What happened?
**PostProcessing/Runtime/PostProcessLayer.cs** 
Errors on line 923, 956, 999: Type 'UnityEngine.MaterialPropertyBlock' does not contain a definition for 'SetInt'.

**PostProcessing/Runtime/RuntimeUtilities.cs**
Errors on line 342, 354: Type 'UnityEngine.Rendering.CommandBuffer' does not contain a definition for 'SetGlobalInt'

I reverted from bleeding edge to the 2.0.9 (latest non-preview) release and the issue is non-existant.

##### Post-processing stack version (v1, v2)?
v2

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
Unity 2017.4.13f1, Windows 10, Standalone Windows.

**Comment by MamaCatDev** [comment]
Yeah, I suspect this is a 2018 change that bled back into 2017 when it shouldn't have.

**Comment by Jonny10** [comment]
You can change 'SetInt' to 'SetFloat' in the offending code to clear these errors.
